src/coreclr/inc/clrconfigvalues.h

```diff
@@ -302,6 +302,7 @@ RETAIL_CONFIG_STRING_INFO(EXTERNAL_JitName, W("JitName"), "Primary Jit to use")
 #if defined(ALLOW_SXS_JIT)
 RETAIL_CONFIG_STRING_INFO(EXTERNAL_AltJitName, W("AltJitName"), "Alternative Jit to use, will fall back to primary jit.")
 RETAIL_CONFIG_STRING_INFO(EXTERNAL_AltJit, W("AltJit"), "Enables AltJit and selectively limits it to the specified methods.")
+RETAIL_CONFIG_STRING_INFO(EXTERNAL_AltJitOs, W("AltJitOS"), "Sets target OS for AltJit or uses native one by default. Only applicable for ARM/AMR64 at the moment.")
```

Comment:
nit: I would use EXTERNAL_AltJitOS so the name exactly matches (including case) the variable name.

Also: typo: AMR64 => ARM64

Reply:
Oops, will fix in a follow up!

Regarding SuperPMI - not sure, I just wanted to test ABI differences on ARM64